src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Initialization.cs

```diff
@@ -41,13 +41,6 @@ static CryptoInitializer()
                 // these libraries will be unable to operate correctly.
                 throw new InvalidOperationException();
             }
-
```

Comment:
Above at line 14, we should be able to delete the libcrypto cctor.  Comment should also be updated.